main.md

```diff
@@ -442,7 +441,14 @@ values. The SVC therefore maps claim names to JSON-encoded arrays.
 
 For transporting the SVC together with the SD-JWT from the issuer to the holder,
 the SVC is base64url-encoded and appended to the SD-JWT using a period character `.` as the
-separator. For Example 1, the combined format looks as follows:
+separator. 
+
+The SVC and SD-JWT are implicitly linked through the hash values of the claims
+in the SVC that is included in the SD-JWT. To ensure that the correct SVC and 
+SD-JWT pairings are being used, the holder should verify the binding between
+SVC and SD-JWT, as defined in the Verification Section of this document.
```

Comment:
I think this sentence works with and without the comma, but without the comma is probably closer to what we want to say.

```suggestion
SVC and SD-JWT as defined in the Verification Section of this document.
```